src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -302,7 +303,7 @@ private async Task DisposeAsyncCore()
 
         private async Task SelectAndStartTransport(TransferFormat transferFormat, CancellationToken cancellationToken)
         {
-            var uri = _httpConnectionOptions.Url;
+            var uri = _httpConnectionOptions.Url!;
```

Comment:
Seen this twice so far. Should this be property be made non-null?

Reply:
Is there a way to make a property non-null when it isn't assigned in the ctor? I guess setting it to string.Empty is a workaround.

Reply:
`public string Url { get; set; } = default!;` is something we do fairly commonly.